src/runtime_strings.js

```diff
@@ -187,7 +187,7 @@ function stringToUTF8Array(str, heap, outIdx, maxBytesToWrite) {
     } else {
       if (outIdx + 3 >= endIdx) break;
 #if ASSERTIONS
-      if (u >= 0x200000) warnOnce('Invalid Unicode code point 0x' + u.toString(16) + ' encountered when serializing a JS string to a UTF-8 string in wasm memory! (Valid unicode code points should be in range 0-0x1FFFFF).');
+      if (u >= 0x110000) warnOnce('Invalid Unicode code point 0x' + u.toString(16) + ' encountered when serializing a JS string to a UTF-8 string in wasm memory! (Valid unicode code points should be in range 0-0x10FFFF).');
```

Comment:
How about `u > 0x10FFFF` so that check matches the comment?

Reply:
I see and think it's better. I fix so. Thank you.